lando/README.md

```diff
@@ -128,7 +128,6 @@ If you want to use [Lando](https://lando.dev/) for local development, here are s
       127.0.0.1 womensleadership.suhumsci.loc
       127.0.0.1 womensleadershipcp.suhumsci.loc
     ```
-
```

Comment:
Can you please restore the line breaks? It doesn't affect the markdown rendering, but makes the source file harder to read.